test/test-growpart

```diff
@@ -51,6 +51,7 @@ truncate --size "$size" "$img"
 
 lodev=$(losetup --show --find --partscan "$img")
 LODEV=$lodev
+udevadm settle
```

Comment:
@blackboxsw this is very similar to what was dropped in https://github.com/canonical/cloud-utils/commit/1520f840f377dc00ddd605652f37113a3e89c154. Just in case you're wondering if it's safe to run `udevadm settle`, we can assume it's present and so on.

Can be tested by preparing a qemu autopkgtest image (`autopkgtest-buildvm-ubuntu-cloud -r hirsute -v`). Can be tested using `autopkgtest-virt-qemu`, but this requires merging this branch into `ubuntu/devel`, which I did locally.

Reply:
Thanks for this test step I felt okay about udevadm settle, but I had a broken autopkgtest setup on my box and spent my time going back and getting my autopkgtest setup working again.  Running autopkgtests now and will +1 afterward